### PR TITLE
Feat/sui widget embedder generate

### PR DESCRIPTION
**Pull request by carlesnunez**
## Description
A boilerplate generator taking as a guidance the one that was made on sui-studio. 

## Related Issue
This PR closes issue #213

## Example
You can test it installing sui-widget-embedder and running the next command:
```
sui-widget-embedder generate <widgetName> -E 'your-path-regexp'
```

### CODE REVIEW

File: packages/sui-widget-embedder/file-templates/_index.scss.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = (sitePrefix) => `@import '~@schibstedspain/${sitePrefix}-theme/lib/index.scss';`
```
Comment (davidbarna) [inline packages/sui-widget-embedder/file-templates/_index.scss.js]: hmmmm an open-source tool that creates references to `@schibstedspain`.
@carlosvillu

File: packages/sui-widget-embedder/bin/sui-widget-embedder-generate.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+const BASE_DIR = process.cwd()
+const WIDGET_DIR = `/widgets/${widget}/`
+const WIDGET_PATH = `${BASE_DIR}${WIDGET_DIR}`
+const WIDGET_ENTRY_JS_POINT_FILE = `${WIDGET_PATH}index.js`
+const WIDGET_PACKAGE_JSON_FILE = `${WIDGET_PATH}package.json`
+const WIDGET_ENTRY_SCSS_POINT_FILE = `${WIDGET_PATH}index.scss`
+const WIDGET_BASE_PROJECT_JSON_POINT_FILE = path.join(process.cwd(), 'package.json')
+const {widgetRegExpIdentifier} = program
+const packageInfo = require(path.join(process.cwd(), 'package.json'))
```
Comment (davidbarna) [inline packages/sui-widget-embedder/bin/sui-widget-embedder-generate.js]: duplicated op `path.join(process.cwd(), 'package.json')`

File: packages/sui-widget-embedder/file-templates/_index.scss.js (hunk 1 above)
Comment (carlosvillu) [reply packages/sui-widget-embedder/file-templates/_index.scss.js]: We are doing the same here
https://github.com/SUI-Components/sui/blob/master/packages/sui-studio/bin/sui-studio-generate.js#L144

We have to define a policy for this kind of thing. Meanwhile I would not change it.

But It is true, that can`t be in an open source repo.

Comment (carlosvillu) [reply packages/sui-widget-embedder/file-templates/_index.scss.js]: We are doing the same here
https://github.com/SUI-Components/sui/blob/master/packages/sui-studio/bin/sui-studio-generate.js#L144

We have to define a policy for this kind of thing. Meanwhile I would not change it.

But It is true, that can`t be at an open source repo.

**Pull request merged**